src/Lykke.Service.FixGateway/Modules/RabbitMqModule.cs

```diff
@@ -49,6 +49,22 @@ protected override void Load(ContainerBuilder builder)
                 .SetMessageDeserializer(new JsonMessageDeserializer<MarketOrderWithTrades>())
                 .SetMessageReadStrategy(new MessageReadWithTemporaryQueueStrategy())
                 .SetLogger(c.Resolve<ILog>()));
+
+
+            var los = _settings.CurrentValue.RabbitMq.IncomingLimitOrders;
+            var limitOrderSettings = new RabbitMqSubscriptionSettings
+            {
+                ConnectionString = los.ConnectionString,
+                ExchangeName = los.Exchange,
+                QueueName = los.Queue,
+                IsDurable = true
+            };
+
+            var loes = new DefaultErrorHandlingStrategy(_log, orderSettings);
```

Comment:
loes? Limit Order Error strategy?

Reply:
Yes. It is temporary variable. It lives only 2 lines. A good name doesn't give anything, but takes time for composing